tests/unit/test-cmd/test.run.js

```diff
@@ -406,5 +407,23 @@ describe('run', () => {
            profileCreateIfMissing: true,
          })
     );
+
+    it('throws error when used without firefox-profile or chromium-profile',
+       async () => {
+         const cmd = prepareRun();
+
+         let actualError = null;
+         try {
+           await cmd.run({ profileCreateIfMissing: true });
+         } catch (error) {
+           actualError = error;
+         }
+
+         assert.instanceOf(actualError, UsageError);
+         assert.match(
+           actualError && actualError.message,
+           /should be paired with --firefox-profile or --chromium-profile/
+         );
```

Comment:
we should use `await assert.isRejected(...)` (provided by the chai-as-promised dev dependency and also used in some of the other tests, e.g. test.adb.js should be one of the test file already using this assertion functions).